### PR TITLE
feat: translate amt sku value to human readable string

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: src/app/devices/general/general.component.html [hunk 1]
```diff
@@ -41,9 +41,7 @@
         <div class="flex flex-50 justify-end">
           @if (!isLoading) {
             <span class="mat-h3" data-cy="amtSku">
-              @if (amtVersion?.CIM_SoftwareIdentity?.responses[6].VersionString) {
-                {{ amtVersion?.CIM_SoftwareIdentity?.responses[6].VersionString }}
-              }
+                {{ amtBuild }}
```
Comment (madhavilosetty-intel) [inline src/app/devices/general/general.component.html:44]: Don't we need the if condition any more?

Comment (madhavilosetty-intel) [inline src/app/devices/general/general.component.html:44]: Don't we need a if condition? I don't see a look up like amtSku. Also, data-cy value should be amtBuild

Comment (matt-primrose) [reply src/app/devices/general/general.component.html:44]: updated!

**Review by madhavilosetty-intel — APPROVED** [review]

**Pull request merged**